### DISCUSSION TITLE
Updated Docker image

### DISCUSSION THREAD

**Discussion by mattdy** [robbrad/UKBinCollectionData#745] [General]
I've tried using the Docker image at [robbrad182/uk-bin-collection](https://hub.docker.com/r/robbrad182/uk-bin-collection), however it hasn't been updated in 5 months, so is missing some fixes in newer releases.

Would it be possible to get a new image auto-built and pushed on each release? 

Thanks in advance!

**Comment by robbrad** [comment]
Done. Made it on-demand and also scheduled for a Sunday

Will look to incorporate the rebuild on release too

**Comment by mattdy** [comment]
Perfect, thanks very much!
